src/status_im/ui/screens/wallet/settings/subs.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im.ui.screens.wallet.settings.subs
+  (:require [re-frame.core :as re-frame]
+            [status-im.utils.ethereum.core :as ethereum]))
+
+;; A set of visible-tokens for current network
+(re-frame/reg-sub :wallet.settings/visible-tokens
+  (fn [{:keys [network] :accounts/keys [current-account-id] :as db}]
+    (let [chain-id (ethereum/network->chain-id network)]
+      (get-in db [:accounts/accounts current-account-id :settings :wallet :visible-tokens chain-id]))))
```

Comment:
This subscription is Level-1, meaning if anything changes in db, it will be unnecessary recomputed.
It should only take what's necessary as inputs (subscriptions to :network and :current-account-id keys) and produce results based on that.
https://github.com/status-im/ideas/issues/31 - user story number 2